tests/test_download.py

```diff
@@ -96,6 +104,87 @@ def test_download_service(
         assert response.status_code == 400, f"{response.data}"
 
 
+@pytest.mark.integration
+@pytest.mark.parametrize(
+    "userdata,erc20_enterprise",
+    [(False, False), ("valid", False), ("invalid", False), (False, True)],
+)
+def test_download_multiple_orders(
+    client, publisher_wallet, consumer_wallet, web3, userdata, erc20_enterprise
+):
+    nft_factory_address = get_data_nft_factory_address(web3)
+    asset1 = get_registered_asset(publisher_wallet, erc20_enterprise=erc20_enterprise)
+    service1 = get_first_service_by_type(asset1, ServiceType.ACCESS)
+    mint_100_datatokens(
+        web3, service1.datatoken_address, consumer_wallet.address, publisher_wallet
+    )
+    approve_tokens(
+        web3, service1.datatoken_address, nft_factory_address, 1, consumer_wallet
+    )
+    # create an asset with two services
+    asset2 = get_registered_asset(
+        publisher_wallet, erc20_enterprise=erc20_enterprise, no_of_services=2
+    )
+    service2 = get_first_service_by_type(asset2, ServiceType.ACCESS)
+    mint_100_datatokens(
+        web3, service2.datatoken_address, consumer_wallet.address, publisher_wallet
+    )
+    approve_tokens(
+        web3, service2.datatoken_address, nft_factory_address, 1, consumer_wallet
+    )
+    # get the 2nd service
+    service3 = get_service_by_index(asset2, 1)
+    mint_100_datatokens(
+        web3, service3.datatoken_address, consumer_wallet.address, publisher_wallet
+    )
+    approve_tokens(
+        web3, service3.datatoken_address, nft_factory_address, 1, consumer_wallet
+    )
+    tx_id, _ = start_multiple_order(
+        web3,
+        [
+            (
+                service1.datatoken_address,
+                consumer_wallet.address,
+                service1.index,
+                get_provider_fees(asset1, service1, consumer_wallet.address, 0),
+                (
+                    "0x0000000000000000000000000000000000000000",
+                    "0x0000000000000000000000000000000000000000",
+                    0,
+                ),
+            ),
+            (
+                service2.datatoken_address,
+                consumer_wallet.address,
+                service2.index,
+                get_provider_fees(asset2, service2, consumer_wallet.address, 0),
+                (
+                    "0x0000000000000000000000000000000000000000",
```

Comment:
I think you can use ZERO_ADDRESS here (and in other places) instead

Reply:
there is no ZERO_ADDRESS defined, so let's create another task for this (very low prio)